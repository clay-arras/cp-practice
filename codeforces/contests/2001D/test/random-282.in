1
16
9 12 3 16 20 13 8 2 20 21 16 24 3 24 4 25
