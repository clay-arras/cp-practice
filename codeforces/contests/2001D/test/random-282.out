12
9 12 20 13 8 2 21 16 24 3 4 25
